mautic/mautic
# Disabled send button from list If sending in background

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This is follow up for already merged PR https://github.com/mautic/mautic/pull/11351
We forgot disable send button from email list view. This PR added it there 



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Setup Publish UP of segment email date lower like now
3. Go to emails page and see send button is disabled with tooltip message for that email 

![image](https://user-images.githubusercontent.com/462477/218004130-83c36c29-0381-4071-9d0b-2012cff7a622.png)
